jobs/Salesforce/readme.md

```diff
@@ -0,0 +1,11 @@
+# Mews SF developer task
+
+The task is to implement an ExchangeRateProvider for Czech National Bank. Find data source on their web - part of the task is to find the source of the exchange rate data and a way how to extract it from there.
+
+The solution has to be buildable, runnable and the program should output the obtained exchange rates.
```

Comment:
Do we want to limit them to using some specific technology (e.g. .net/js)? If not - we should just mention - that our goal is to understand their programming skills, but as SF developers, it's clear that they will use other constructs to avoid some misunderstandings.